app/guests/[guestId].tsx

```diff
@@ -0,0 +1,268 @@
+import { Ionicons } from '@expo/vector-icons';
+import { Image } from 'expo-image';
+import { router, useFocusEffect, useLocalSearchParams } from 'expo-router';
+import { useCallback, useState } from 'react';
+import {
+  Pressable,
+  StyleSheet,
+  Switch,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import placeholder from '../../assets/candidate-default.avif';
+import { colors } from '../../constants/colors';
+import type { Guest } from '../../migrations/00000-createTableGuests';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: colors.background,
+    paddingLeft: 10,
+    paddingRight: 10,
+  },
+  avatar: {
+    width: 200,
+    height: 200,
+    borderRadius: 100,
+    overflow: 'hidden',
+    marginBottom: 16,
+  },
+  avatarImage: {
+    width: '100%',
+    height: '100%',
+  },
+  text: {
+    textAlign: 'center',
+    fontSize: 24,
+    fontFamily: 'Poppins_700Bold',
+    color: colors.text,
+  },
+  textContainer: {
+    alignItems: 'center',
+    gap: 12,
+  },
+  textSecondary: {
+    textAlign: 'center',
+    fontSize: 14,
+    fontFamily: 'Poppins_400Regular',
+    color: colors.textSecondary,
+  },
+  iconContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: 80,
+    gap: 200,
+  },
+  icon: {
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  button: {
+    marginTop: 30,
+    backgroundColor: colors.text,
+    padding: 12,
+    borderRadius: 12,
+    shadowColor: colors.white,
+    shadowOffset: { width: 0, height: 2 },
+    shadowOpacity: 0.2,
+    shadowRadius: 4,
+    width: '100%',
+  },
+  buttonText: {
+    fontFamily: 'Poppins_400Regular',
+    color: colors.cardBackground,
+    textAlign: 'center',
+    fontSize: 18,
+  },
+  label: {
+    fontSize: 18,
+    fontFamily: 'Poppins_400Regular',
+    color: colors.text,
+    marginBottom: 8,
+  },
+  input: {
+    color: colors.text,
+    backgroundColor: colors.background,
+    borderColor: colors.textSecondary,
+    borderWidth: 1,
+    borderRadius: 8,
+    padding: 8,
+    marginBottom: 16,
+    fontSize: 16,
+    fontFamily: 'Poppins_400Regular',
+  },
+  inputFocused: {
+    borderColor: colors.white,
+  },
+  inputContainer: {
+    width: '100%',
+    backgroundColor: colors.cardBackground,
+    borderRadius: 12,
+    padding: 12,
+    marginBottom: 16,
+  },
+});
+
+export default function GuestPage() {
+  const { guestId } = useLocalSearchParams();
+
+  const [edit, setEdit] = useState<boolean>(false);
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [attending, setAttending] = useState<boolean>(false);
```

Comment:
these all don't need generic args
```suggestion
  const [edit, setEdit] = useState(false);
  const [firstName, setFirstName] = useState('');
  const [lastName, setLastName] = useState('');
  const [attending, setAttending] = useState(false);
```

Reply:
added in this commit 
[02412d3](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/02412d3489fef1d658e479ba88bfe6f07a8187a7)